model.py

```diff
@@ -0,0 +1,151 @@
+
+    userpair_relationship = db.relationship('UserPair')
+    communitypair_relationship = db.relationship('CommunityPair')
+
+    def __repr__(self):
+        return f'<Wine Information wine_id={self.wine_id} wine_name={self.wine_name}>'
+
+
+class Cheese(db.Model):
+    """A cheese."""
+
+    __tablename__ = "cheeses"
+
+    cheese_id = db.Column(db.Integer,
+                          primary_key=True,
+                          autoincrement=True,)
+    cheese_name = db.Column(db.String, nullable=False)
+    cheese_img = db.Column(db.String)
+    cheese_region = db.Column(db.String, nullable=False)
+    cheese_description = db.Column(db.Text, nullable=False)
+    cheese_bio = db.Column(db.Text, nullable=False)
+    cheese_animal = db.Column(db.String)
+    cheese_density = db.Column(db.String, nullable=False)
+
+    userpair_relationship = db.relationship('UserPair')
+    communitypair_relationship = db.relationship('CommunityPair')
+
+    def __repr__(self):
+        return f'<Cheese Information cheese_id={self.cheese_id} cheese_name={self.cheese_name}>'
+
+
+class UserPair(db.Model):
+    """A user's created paired wine and cheese."""
+
+    __tablename__ = "user_pairs"
+
+    userpair_id = db.Column(db.Integer,
+                            primary_key=True,
+                            autoincrement=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.user_id'))
+    wine_id = db.Column(db.Integer, db.ForeignKey('wines.wine_id'))
+    cheese_id = db.Column(db.Integer, db.ForeignKey('cheeses.cheese_id'))
+
+    user_relationship = db.relationship('User')
+    wine_relationship = db.relationship('Wine')
+    cheese_relationship = db.relationship('Cheese')
+
+    def __repr__(self):
+        return f'UserPair userpair_id={self.userpair_id} user_id={self.user_id}'
+
+
+class CommunityPair(db.Model):
+    """An already established cheese and wine pairing that can be seen viewed/accessed by users."""
+
+    __tablename__ = "community_pairs"
+
+    communitypair_id = db.Column(db.Integer,
+                                 primary_key=True,
+                                 autoincrement=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.user_id'))
+    wine_id = db.Column(db.Integer, db.ForeignKey(wines.'wine_id'))
+    cheese_id = db.Column(db.Integer, db.ForeignKey(cheeses.'cheese_id'))
+
+    user_relationship = db.relationship('User')
+    wine_relationship = db.relationship('Wine')
+    cheese_relationship = db.relationship('Cheese')
+
+    def __repr__(self):
+        return f'Community Pair communitypair_id={self.communitypair_id} user_id={self.user_id}'
+
+
+class
+
+
+class Rating(db.Model):
+    """A rating that a user can create and publish on their profile"""
+
+    __tablename__ = "ratings"
+
+    rating_id = db.Column(db.Integer,
+                          primary_key=True,
+                          autoincrement=True)
+    pairtype_id = db.Column(db.String, db.ForeignKey('pair_types.pairtype_id'))

```

Comment:
Is this supposed to be a foreign key for 'pair_types.pairtype_id' on line 121? where is this coming from?